### PR TITLE
Home page for ecommerce

### PR DESCRIPTION
**Pull request by vashnavichauhan1825**

### CODE REVIEW

**Review by b0llu — APPROVED** [review]
If your only adding images you can merge it without PR just add a comment in the PR saying its images only

File: main.css [hunk 1]
```diff
@@ -1,14 +1,193 @@
...
+     display:block;
+ }
+
+ /* contact nav */
+
+ .contact-nav{
+    display: flex;
+    justify-content: space-between;
+    align-items:center;
+    padding: 7px 179px 8px;
```
Comment (MayankKumar10) [inline main.css:19]: use rem instead of px

File: main.css [hunk 2]
```diff
@@ -1,14 +1,193 @@
...
+    border-bottom: 1px solid var(--text-color);
+}
+
+
+ .icon-button{
+    display: flex;
```
Comment (MayankKumar10) [inline main.css:37]: make a helper class for flex

File: main.css [hunk 3]
```diff
@@ -1,14 +1,193 @@
...
+
+.search-nav .search-cont{
+    display: flex;
+    gap: 20px;
+    align-items: center;
+}
+.search-nav select{
+    background-color: transparent;
+    border: 1px solid var(--text-color);
+    padding: 8px 8px;
```
Comment (MayankKumar10) [inline main.css:132]: Don't use two values in shorthand if they are the same

File: main.css [hunk 4]
```diff
@@ -1,14 +1,193 @@
...
+    border-radius: 5px;
+}
+.search-nav option{
+    background-color: transparent;
+    border: 1px solid var(--text-color);
+    padding: 8px 8px;
+    border-radius: 5px;
```
Comment (MayankKumar10) [inline main.css]: make this is helper class so don't need to repeat it

File: main.css [hunk 5]
```diff
@@ -1,14 +1,193 @@
...
+    padding: 5px;
+    font-size: 1.3rem;
+    border-radius:5px;
+}
+
+
+
+.badge .icon-bell{
+    background-color:#3a7ce3;
+    top: -19%;
```
Comment (MayankKumar10) [inline main.css:169]: don't use negative values instead here you can use bottom 19%

File: main.css [hunk 6]
```diff
@@ -190,4 +194,70 @@ li:hover{
...
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.feature-cont{
+    /* background-color: blue; */
+    width: auto;
+    height: 400px;
```
Comment (MayankKumar10) [inline main.css]: use rem here instead of px

File: main.css [hunk 7]
```diff
@@ -190,4 +194,70 @@ li:hover{
...
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    gap: 19px;
+}
+
+.product-display{
+    border: 1px solid var(--text-color);
+    border-radius:5px;
+    height: 230px;
+    width: 180px;
```
Comment (MayankKumar10) [inline main.css:259]: use rem instead of px

File: main.css [hunk 8]
```diff
@@ -258,6 +262,155 @@ li:hover{
...
-    margin-bottom:3rem;
+    margin-left: 10rem;
+    margin-bottom: 3rem;
+    margin-top: 8rem;
+}
+
+/* special feature */
+
+.feature-second{
+    display: flex;
```
Comment (MayankKumar10) [inline main.css:284]: make more helper classes for flex so you don't have to repeat it

File: main.css [hunk 9]
```diff
@@ -258,6 +262,155 @@ li:hover{
...
+
+
+.flex-center-product{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.popular-category ul{
+    list-style: none;
```
Comment (MayankKumar10) [inline main.css:398]: make helper class ul and li and properties like list-style: none and more

File: main.css [hunk 10]
```diff
@@ -258,6 +262,155 @@ li:hover{
...
+}
+
+.popular-item i{
+    font-size:20px;
+}
+
+/* popular review */
+
+.popular-category .review{
+    background-color: turquoise;
```
Comment (MayankKumar10) [inline main.css]: If you used this color elsewhere you can define it in the root directory

File: main.css [hunk 11]
```diff
@@ -46,21 +47,25 @@
...
+.head-margin{
+   margin-right:5rem;
+}
+
 
 
-ul{
+.category-nav ul{
     list-style: none;
     margin-top: 417px;
```
Comment (MayankKumar10) [inline main.css:77]: Don't use big values in the margin it will affect your site in the longer run.

File: main.css [hunk 12]
```diff
@@ -258,6 +262,155 @@ li:hover{
 
 .heading-product{
     display: inline-block;
-    margin-left:10rem;
-    margin-bottom:3rem;
+    margin-left: 10rem;
+    margin-bottom: 3rem;
+    margin-top: 8rem;
```
Comment (MayankKumar10) [inline main.css:278]: shorthand property of margin and padding in Top, Right, Bottom, and Left use that in that order

File: main.css [hunk 13]
```diff
@@ -125,7 +125,7 @@ li:hover{
     padding: 8px 8px;
```
Comment (MayankKumar10) [inline main.css:138]: use one value here it work fine

File: main.css [hunk 14]
```diff
@@ -431,4 +434,33 @@ li:hover{
...
     width: 15rem;
     display: inline-block;
+}
+
+/* footer  */
+
+footer{
+    display: grid;
+    grid-template-columns: repeat(4,1fr);
+    height: 260px;
```
Comment (MayankKumar10) [inline main.css]: use rem here

File: main.css [hunk 15]
```diff
@@ -3,7 +3,7 @@
 
 
  body{
-     background-color:black;
+     background-color:var(--secondaryDark);
      margin:0;
```
Comment (MayankKumar10) [inline main.css]: add top0 , left0 in body tag

**Review by MayankKumar10 — APPROVED** [review]
Just Suggesting some corrections I found, Everything else looks good.

File: main.css (hunk 3 above)
Comment (vashnavichauhan1825) [reply main.css:132]: yes sure

**Pull request merged**